constants/src/main/java/org/jboss/pnc/enums/JobNotificationProgress.java

```diff
@@ -18,12 +18,20 @@
 package org.jboss.pnc.enums;
 
 /**
- *
+ * Enum describing job progress in notifications.
  * @author Honza Brázdil &lt;jbrazdil@redhat.com&gt;
  */
-
 public enum JobNotificationProgress {
+    /**
+     * The job is waiting. For example build waiting for dependencies.
+     */
     PENDING,
+    /**
+     * The job is running. For example build is building.
+     */
     IN_PROGRESS,
+    /**
+     * The job has finished. For example build failed.
```

Comment:
You're clearly a glass half full guy :)

Reply:
Well, I was thinking that example "build succeeded" was obvious and that it can represent also a failure was less obvious :)